### PR TITLE
Ignore submodules in coverage reports

### PR DESCRIPTION
**Pull request by Geod24**
This should massively speed up upload and improve coverage.

### CODE REVIEW

**Review by AndrejMitrovic — APPROVED** [review]

**Comment by Geod24** [comment]
Hold on, it doesn't work

File: .codecov.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ignore:
+  - "*-agora-submodules-*"
```
Comment (AndrejMitrovic) [inline .codecov.yaml]: I think this should be just `submodules-*.lst`? The files seem to start with `submodules` from what I can tell.

Comment (AndrejMitrovic) [reply .codecov.yaml]: E.g.:

```
-rw-------    1 andrejmitrovic  staff   3.2K Oct 24 18:20 source-scpd-types-Utils.lst
-rw-------    1 andrejmitrovic  staff   3.6K Oct 24 18:20 source-scpd-types-XDRBase.lst
-rw-------    1 andrejmitrovic  staff   2.3K Oct 24 18:20 submodules-vibe-core-source-vibe-appmain.lst
-rw-------    1 andrejmitrovic  staff   2.0K Oct 24 18:20 submodules-ocean-src-ocean-LibFeatures.lst
-rw-------    1 andrejmitrovic  staff   5.2K Oct 24 18:20 submodules-ocean-src-ocean-core-Array.lst
```

**Comment by Geod24** [comment]
I don't think this will work, and it's not super important either.

**Pull request closed without merging**